### PR TITLE
Updater russian locale

### PR DESCRIPTION
**Pull request by henbe**
In a doubt if spell names in square brackets could be translated. (lines 14, 19, 80)

### CODE REVIEW

File: BattleOfDazaralor/Locales/ruRU.lua [hunk 1]
```diff
@@ -5,104 +5,104 @@ if L then
...
+	L.custom_on_fixate_plates_desc = "Отображаеть иконку на Полосе Здоровья противнка, что преследует Вас.\nНеобходимо включить Полосы Здоровья противников. Данная опция поддерживает только KuiNameplates."
 
-	--L.absorb = "Absorb"
-	--L.absorb_text = "%s (|cff%s%.0f%%|r)"
-	--L.cast = "Cast"
-	--L.cast_text = "%.1fs (|cff%s%.0f%%|r)"
+	L.absorb = "Щит"
+	L.absorb_text = "%s (|cff%s%.0f%%|r)"
+	L.cast = "Произнесение"
+	L.cast_text = "%.1fs (|cff%s%.0f%%|r)"
```
Comment (funkydude) [inline BattleOfDazaralor/Locales/ruRU.lua]: Are you sure this shouldn't be `%.1fс` ?

**Comment by funkydude** [comment]
Do you plan on translating the other files?
```
Locales/ruRU.lua
Uldir/Locales/ruRU.lua
Azeroth/Locales/ruRU.lua
Plugins/Locales/ruRU.lua
```

File: BattleOfDazaralor/Locales/ruRU.lua (hunk 1 above)
Comment (henbe) [reply BattleOfDazaralor/Locales/ruRU.lua]: It can stay as it is, not a big deal. Just changed back because over timers use an "s" mostly.

**Comment by henbe** [comment]
> Do you plan on translating the other files?
> 
> ```
> Locales/ruRU.lua
> Uldir/Locales/ruRU.lua
> Azeroth/Locales/ruRU.lua
> Plugins/Locales/ruRU.lua
> ```

Yes.

**Pull request closed without merging**